### PR TITLE
Azure theme style updates across several components 

### PR DESCRIPTION
**Pull request by Jacqueline-ms**
13px across various components, detailslist, datepicker, commandbar, & label style updates

#### Pull request checklist

- [X] Addresses an existing issue: style bugs
- [X] Include a change request file using `$ yarn change`

#### Description of changes

- detailsList checkmarks
- datepicker disabled state
- command bar height / border color
- label color
- 13px font on multiple components

### CODE REVIEW

File: packages/azure-themes/src/azure/AzureThemeDark.ts [hunk 1]
```diff
@@ -8,7 +8,7 @@ const darkExtendedSemanticColors: Partial<IExtendedSemanticColors> = {
   commandBarBorder: DarkSemanticColors.commandBar.border,
   bodyBackground: DarkSemanticColors.background,
   bodyText: DarkSemanticColors.text.body,
-  bodyTextHovered: DarkSemanticColors.text.bodyHovered,
+  bodyTextHover: DarkSemanticColors.text.bodyHovered,
```
Comment (hyoshis) [inline packages/azure-themes/src/azure/AzureThemeDark.ts:11]: Sorry, can you change it back to bodyTextHovered? 

I looked around the codebase and it looks like the following format is used for hover variable naming:
- xxxHovered
- xxxOnHover

So either bodyTextHovered or bodyTextOnHover would work. For this case, we can keep the original name.

Comment (Jacqueline-ms) [reply packages/azure-themes/src/azure/AzureThemeDark.ts:11]: Got it, thank you

File: packages/azure-themes/src/stories/components/tags.tsx [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+import { Checkbox, ICheckboxStyles } from 'office-ui-fabric-react/lib/Checkbox';
+import { mergeStyles } from 'office-ui-fabric-react/lib/Styling';
+
+const rootClass = mergeStyles({
+  maxWidth: 500,
+});
+
+const checkboxStyles: Partial<ICheckboxStyles> = { root: { margin: '10px 0' } };
+
+const _testTags: ITag[] = [
```
[the remark this answers is not in the document]
Comment (Jacqueline-ms) [reply packages/azure-themes/src/stories/components/tags.tsx:13]: This came from Fluent's component but I removed underscore

File: packages/azure-themes/src/stories/components/dateBoundary.tsx [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+  goToToday: 'Go to today',
+  prevMonthAriaLabel: 'Go to previous month',
+  nextMonthAriaLabel: 'Go to next month',
+  prevYearAriaLabel: 'Go to previous year',
+  nextYearAriaLabel: 'Go to next year',
+  closeButtonAriaLabel: 'Close date picker',
+
+  isRequiredErrorMessage: 'Field is required.',
+
```
[the remark this answers is not in the document]
Comment (Jacqueline-ms) [reply packages/azure-themes/src/stories/components/dateBoundary.tsx:40]: Done

File: packages/azure-themes/src/stories/components/dateBoundary.tsx [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+  invalidInputErrorMessage: 'Invalid date format.',
+
```
[the remark this answers is not in the document]
Comment (Jacqueline-ms) [reply packages/azure-themes/src/stories/components/dateBoundary.tsx:42]: Done

File: packages/azure-themes/src/stories/Themes/Themes.tsx [hunk 5]
```diff
@@ -1,3 +1,103 @@
...
+//   Stack,
+//   Fabric,
+//   CompoundButton,
+//   Checkbox,
+//   DatePicker,
+//   SearchBox,
+//   Link,
+// } from 'office-ui-fabric-react';
+// import { AzureCustomizationsLight, AzureCustomizationsDark } from '../../index';
+// import { ButtonCommandBarExample } from '../components/commandBarButton';
```
[the remark this answers is not in the document]
Comment (Jacqueline-ms) [reply packages/azure-themes/src/stories/Themes/Themes.tsx:16]: Done

File: packages/azure-themes/src/azure/AzureThemeDark.ts (hunk 1 above)
Comment (Jacqueline-ms) [reply packages/azure-themes/src/azure/AzureThemeDark.ts:11]: Complete

**Pull request merged**